src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -321,6 +321,17 @@ TransactionManager::submit_transaction_direct(
   });
 }
 
+seastar::future<> TransactionManager::flush(OrderingHandle &handle)
+{
+  return handle.enter(write_pipeline.ool_writes
```

Comment:
Should `flush()` also need to enter `write_pipeline.reserve_projected_usage` ?

Reply:
Yep, good catch.  Repushed.